tests/specs/spa/capture-interactions.e2e.js

```diff
@@ -0,0 +1,31 @@
+describe('spa captures interaction when', () => {
+  it('hashchange fires after finish', async () => {
+    const url = await browser.testHandle.assetURL('spa/hashchange-onclick.html')
+    await Promise.all([
+      browser.testHandle.expectInteractionEvents(), // Discard the initial page load interaction
```

Comment:
not necessary here, but we should consider adding an argument to this expect to select for the trigger type